portal/server/routes/contract/bond/index.js

```diff
@@ -490,33 +415,22 @@ router.post('/contract/:_id/bond/:bondId/confirm-requested-cover-start-date', as
         dateOfCoverChange: getNowAsEpoch().toString(),
       };
 
-      const { bond, validationErrors } = await postToApi(
-        api.updateBondCoverStartDate(
-          dealId,
-          bondId,
-          newBondDetails,
-          userToken,
-        ),
-        errorHref,
-      );
+      const { bond, validationErrors } = await postToApi(api.updateBondCoverStartDate(dealId, bondId, newBondDetails, userToken), errorHref);
 
       requestedCoverValidationErrors = {
         ...validationErrors,
       };
       bondToRender = bond;
     }
 
-    if (!requestedCoverValidationErrors.errorList
-      || (requestedCoverValidationErrors.errorList
-          && !requestedCoverValidationErrors.errorList.requestedCoverStartDate)) {
+    if (
+      !requestedCoverValidationErrors.errorList ||
+      (requestedCoverValidationErrors.errorList && !requestedCoverValidationErrors.errorList.requestedCoverStartDate)
+    ) {
       addFacilityToSessionConfirmedStartDates();
     }
 
-    if (
-      requestedCoverValidationErrors
-      && requestedCoverValidationErrors.errorList
-      && requestedCoverValidationErrors.errorList.requestedCoverStartDate
-    ) {
+    if (requestedCoverValidationErrors && requestedCoverValidationErrors.errorList && requestedCoverValidationErrors.errorList.requestedCoverStartDate) {
```

Comment:
Can we use shorter form `requestedCoverValidationErrors?.errorList?.requestedCoverStartDate`

Reply:
Have updated.